### PR TITLE
1.0.1 Doc Changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Fixed, typo, structure, and other information flow issues
- Added annotated SS
- Overview and step-by-step instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Release Information
 
-- **Version**: 1.0.1 <sup>Preview</sup>
+- **Version**: 1.0.1![preview icon](./docs/res/icon-preview.svg)
 
 - **Certified**: No
```
Comment (cs-dev-chaudhary) [inline README.md:5]: Certified: Yes

File: .gitignore [hunk 2]
```diff
@@ -157,4 +157,5 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
```
Comment (cs-dev-chaudhary) [inline .gitignore:157]: Release_note.md missing.

File: docs/setup.md [hunk 3]
```diff
@@ -3,23 +3,34 @@
...
 # Installation
 1. To install a widget, click **Content Hub** > **Discover**.
-2. From the list of widget that appears, search for and select **MITRE ATT&CK Alert Incident Spread Widget**.
-3. Click the **MITRE ATT&CK Alert Incident Spread Widget** widgetcard.
-4. Click **Install** on the bottom to begin installation.
+2. From the list of widget that appears, search for **MITRE ATT&CK Alert Incident Spread Widget**.
+3. Click the **MITRE ATT&CK Alert Incident Spread Widget** widget card.
+4. Click **Install** on the lower part of the screen to begin installation.
+
+# Prerequisites
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md:10]: Would you also like to add recommended section to promote MITRE Framework SP?

File: docs/res/detail_view.png
Comment (cs-dev-chaudhary) [inline docs/res/detail_view.png:1]: Please check the details view.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ## Release Information
 
-- **Version**: 1.0.1 <sup>Preview</sup>
+- **Version**: 1.0.1![preview icon](./docs/res/icon-preview.svg)
```
Comment (cs-dev-chaudhary) [inline README.md:3]: Need to remove preview i guess.

**Pull request closed without merging**